.travis.yml

```diff
@@ -5,6 +5,7 @@ python:
   - "3.2"
   - "3.3"
   - "3.4"
+  - "3.6"
```

Comment:
Should we go bolder and try 3.11?

Reply:
> We are unable to start your build at this time. You exceeded the number of users allowed for your plan. Please review your plan details and follow the steps to resolution.

@ftomassetti Do you perhaps have access to this https://app.travis-ci.com/github/Mindwerks/pyplatec/requests?requestId=1138733453 to trigger a build?

Reply:
drats... it appears that only personal github accounts can be built on travis, you have to pay for organisations.

Reply:
travis is bust, so I'll investigate github actions next

Reply:
I confirm it is not working for me neither:
![Uploading Screenshot 2025-03-24 at 13.54.40.png…]()